src/Symfony/Component/Translation/Translator.php

```diff
@@ -401,10 +410,20 @@ protected function computeFallbackLocales($locale)
             $locales[] = $fallback;
         }
 
+        $parentLocale = $parentLocales[$locale] ?? null;
+
+        if ('root' === $parentLocale) {
```

Comment:
this is a BC break, as it removes some of the existing fallbacks.

Reply:
Had a quick go and looks possible to keep track of deprecated fallbacks.

But this does raise a question of what should happen when the data changes, eg [CLDR 27](http://cldr.unicode.org/index/downloads/cldr-27) (eg `en_JE` switched from falling back from `en_GB` to `en_001` directly).

Reply:
I think we should follow CLDR